Test/dafny0/AutoContracts.dfy.expect

```diff
@@ -581,11 +581,11 @@ AutoContracts.dfy(50,4): Error: A postcondition might not hold on this return pa
 AutoContracts.dfy(49,24): Related location: This is the postcondition that might not hold.
 AutoContracts.dfy(79,21): Error: A postcondition might not hold on this return path.
 AutoContracts.dfy(60,16): Related location: This is the postcondition that might not hold.
-AutoContracts.dfy[N1](65,20): Related location
 AutoContracts.dfy(79,21): Error: A postcondition might not hold on this return path.
 AutoContracts.dfy(60,16): Related location: This is the postcondition that might not hold.
 AutoContracts.dfy[N1](65,20): Related location
 AutoContracts.dfy(79,21): Error: A postcondition might not hold on this return path.
 AutoContracts.dfy(60,16): Related location: This is the postcondition that might not hold.
+AutoContracts.dfy[N1](65,20): Related location
```

Comment:
Not related to this PR, but why do we have "N1" as a line number?

Reply:
No idea! :)

Reply:
Oh I actually missed that it was in addition to the actual source location. It's the first time I've encountered that.